oracle/datadog_checks/oracle/oracle.py

```diff
@@ -86,10 +89,11 @@ def _get_config(self, instance):
         jdbc_driver = instance.get('jdbc_driver_path')
         tags = instance.get('tags') or []
         custom_queries = instance.get('custom_queries', [])
+        only_custom_queries = instance.get('only_custom_queries')
```

Comment:
```suggestion
        only_custom_queries = instance.get('only_custom_queries', False)
```

Reply:
Oops will fix and look at unit test. Should not have used find /replace.

Reply:
Oops will fix and look at unit test. Should not have used find /replace.